doc/index.rst

```diff
@@ -71,6 +71,11 @@ Features
 * Provides a host of visualization tools for ease of verifying correctness.
 
 
+Community
+---------
+
+You can start discussions and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`__.
```

Comment:
```suggestion
You can start discussions and connect with our community members on our `Discussions Forum <https://discuss.pennylane.ai/c/flamingpy>`_.
```

Reply:
see my comment [here](https://github.com/XanaduAI/flamingpy/pull/96/files#r904210096)

Reply:
@sduquemesa, a single underscore should be there, we possibly have a double there by accident. Although, I would expect the extra symbols to be harmless. I'll accept the suggestion either way.